Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
+}
+
+.bocCheckBox input + span::before
+{
+  margin: 2px;
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  width: 16px;
+  height: 16px;
   display: inline-block;
-  white-space: nowrap;
+  content: "";
+  z-index: 1;
+  position: relative;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input[checked] + span::before
 {
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
 }
 
-.bocCheckBox.disabled
+.bocCheckBox,
+.bocCheckBox > input + span
 {
+  display: flex;
+  align-items: center;
 }
 
-.bocCheckBox input
+/* Fixes for element focus */
+
+.bocCheckBox span[role=checkbox]:focus
 {
-  margin-left: 4px;
-  margin-right: 3px;
-  padding: 0px;
-  vertical-align: -0.1em;
+  outline: none;
 }
 
-.bocCheckBox span + span
+.blink .bocCheckBox span[role=checkbox]:focus > img,
+.blink .bocCheckBox input:focus + span::before
 {
-  vertical-align: middle;
+  box-shadow: 0 0 0 1px black;
+  border-radius: 1px;
 }
 
-.bocCheckBox img
+.mozilla .bocCheckBox span[role=checkbox]:focus > img,
+.mozilla .bocCheckBox input:focus + span::before
 {
-  margin-left: 0px;
-  margin-right: 0;
-  padding: 0px;
-  vertical-align: middle;
-  height: 16px;
-  width: 16px;
+  outline: 1px dotted black;
```

Comment:
There is a follow up ticket for fixing this iirc. I put minimal effort into the outline in this PR.

Reply:
Hmm...yes there is, kinda. Since you did a bit for blink, we should compromise on mostly-minimal effort:
Mozilla
  outline: 2px solid RoyalBlue;
  border-radius: var(--border);
  
 And please make css properties for the outlines in chromium and mozilla. that way we don't have duplication.